lib/node_modules/@stdlib/array/base/accessor/benchmark/benchmark.js

```diff
@@ -23,13 +23,14 @@
 var bench = require( '@stdlib/bench' );
 var Float64Array = require( '@stdlib/array/float64' );
 var randu = require( '@stdlib/random/base/randu' );
+var format = require( '@stdlib/string/format' );
 var pkg = require( './../package.json' ).name;
 var AccessorArray = require( './../lib' );
 
 
 // MAIN //
 
-bench( pkg, function benchmark( b ) {
+bench( format( '%s', pkg ), function benchmark( b ) {
```

Comment:
```suggestion
bench( pkg, function benchmark( b ) {
```

Not necessary to use string interpolation here, as we aren't using concatenation.